lib/aiken/transaction/governance.ak

```diff
@@ -0,0 +1,89 @@
+use aiken/dict.{Dict}
+use aiken/math/rational.{Rational}
+use aiken/transaction/credential.{
+  ColdCommitteeCredential, Credential, DRepCredential, HotCommitteeCredential,
+  PubKeyHash, ScriptHash,
+}
+use aiken/transaction/value.{Lovelace}
+
+pub type DRep {
+  DRep(DRepCredential)
+  DRepAlwaysAbstain
+  DRepAlwaysNoConfidence
+}
+
+pub type Delegatee {
+  DelegStake(PubKeyHash)
+  DelegVote(DRep)
+  DelegStakeVote(PubKeyHash, DRep)
```

Comment:
```suggestion
  DelegateStake(PubKeyHash)
  DelegateVote(DRep)
  DelegateBoth(PubKeyHash, DRep)
```

Reply:
No need for prefixing constructor variants. Namespaces do not clash across types in Aiken.